File: dotnet/src/Pactman.NonprofitCheckPlus/Exceptions/PactmanAuthorizationException.cs
namespace Pactman.NonprofitCheckPlus.Exceptions
{
    /// <summary>HTTP 403. The key is valid but lacks access to the resource.</summary>
    public sealed class PactmanAuthorizationException : PactmanApiException
    {
        /// <summary>Initializes the exception from response metadata.</summary>
        /// <param name="message">Human-readable description of the failure.</param>
        /// <param name="init">The metadata the transport collected from the response.</param>
        public PactmanAuthorizationException(string message, ApiErrorInit init)
            : base(message, init, ErrorCategory.Authorization)
        {
        }
    }
}

File: java/sdk/src/main/java/org/pactman/nonprofitcheckplus/exceptions/PactmanAuthorizationException.java
package org.pactman.nonprofitcheckplus.exceptions;

/** HTTP 403. */
public class PactmanAuthorizationException extends PactmanApiException {

    private static final long serialVersionUID = 1L;

    /**
     * Creates the exception.
     *
     * @param message the message to surface.
     * @param init    the response metadata.
     */
    public PactmanAuthorizationException(String message, ApiErrorInit init) {
        super(message, init, ErrorCategory.AUTHORIZATION);
    }
}
